main.py

```diff
@@ -128,20 +130,21 @@ def start_driver(random_city):
 def generate_account(driver, fake_identity):
     # make fake account info and fill
 
+    info = ''
     email = fake.free_email()
     password = fake.password()
+
     for key in XPATHS_2.keys():
-        match key:
-            case 'email' | 'email-retype':
-                info = fake_identity['email']
-            case 'pass' | 'pass-retype':
-                info = password
-            case 'first_name':
-                info = fake_identity['first_name']
-            case 'last_name':
-                info = fake_identity['last_name']
-            case 'pn':
-                info = fake.phone_number()
+        if key[:5] == 'email':
+            info = fake_identity['email']
+        elif key[:4] == 'pass':
+            info = password
```

Comment:
Minor comment: Can we keep the `or` logic instead of accepting all strings that start a certain way? This may lead to some confusion if more keys are added to `XPATHS_2` and match when they shouldn't.

Reply:
@ChrisCoe fair enough, I'll do this instead of @kpcyrd's suggestion for future compatibility.